index.html

```diff
@@ -207,5 +207,44 @@ <h1>About me</h1>
             </ul>
         </div>
     </div>
+    <form action="https://formspree.io/f/mdovrwgl" method="post">
+        <span>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</span>
+        <ul>
+          <li>
+            <label for="name"></label>
+            <input type="text" id="name" name="user_name" placeholder="Full name"/>
+          </li>
+          <li>
+            <label for="mail"></label>
+            <input type="email" id="mail" name="user_email" placeholder="Email address"/>
+          </li>
+          <li>
+            <label for="msg"></label>
+            <textarea id="msg" name="user_message" placeholder="Write me something..."></textarea>
+          </li>
+          <li class="button">
+            <button type="submit">Get in touch</button>
+          </li>
+        </ul>
+    </form>
+    <footer>
+        <div class="bottomNav">
+            <a href="https://github.com/ivangonzalez224">
+                <img src="images/gitFooter.png" alt="">
+            </a>
+            <a href="https://www.linkedin.com/in/iv%C3%A1n-gonzalez-robles-957491275/">
+                <img src="images/linkFooter.png" alt="">
+            </a>
+            <a href="https://www.angellist.com/">
+                <img src="images/angelistFooter.png" alt="">
+            </a>
+            <a href="https://twitter.com/ivang2204">
+                <img src="images/twitterFooter.png" alt="">
+            </a>
+            <a href="https://medium.com/">
+                <img src="images/mediumFooter.png" alt="">
+            </a>
+        </div>
```

Comment:
- [ ] Kindly consider using a `ul` and `li` tag to list out your social media icons as this will make your code semantically correct and improve the accessibility of your website. Example: 
```
<ul>
   <li>
      <a href="">place your icons here</a>
   </li>
   <li>
      <a href="">place your icons here</a>
   </li>
</ul>
```